database/abuseemail.go

```diff
@@ -180,10 +193,18 @@ func (a AbuseEmail) String() string {
 
 	// write response template
 	sb.WriteString("\nResponse Template:\n\n")
-	sb.WriteString(a.response())
+	sb.WriteString(a.Response())
 	return sb.String()
 }
 
+// Success is a small helper method that indicates whether the abuse email was
```

Comment:
nit: We don't need to include the text `is a small helper method that` - it doesn't really tell us anything.
I know we've been doing that forever but we should stop.